README.rst

```diff
@@ -207,9 +208,11 @@ Middleware settings
 Provider settings
 -----------------
 
-- ``AUTOEXTRACT_USER`` [optional] is your AutoExtract API key. Defaults to ``SCRAPINGHUB_AUTOEXTRACT_KEY`` environment variable.
+- ``AUTOEXTRACT_USER`` [optional] is your AutoExtract API key. If not set, it is
+  taken from SCRAPINGHUB_AUTOEXTRACT_KEY environment variable.
 - ``AUTOEXTRACT_URL`` [optional] the AutoExtract service url. Defaults to the official AutoExtract endpoint.
 - ``AUTOEXTRACT_MAX_QUERY_ERROR_RETRIES`` [optional] Max number of retries for Query-level errors. Defaults to ``3``.
+- ``AUTOEXTRACT_CONCURRENT_REQUESTS_PER_DOMAIN`` [optional] Max number of concurrent requests per domain. If not set, the provider will search for the `CONCURRENT_REQUESTS_PER_DOMAIN` setting instead. Defaults to ``8``.
```

Comment:
Nitpick: this setting defaults to ``CONCURRENT_REQUESTS_PER_DOMAIN``, not to 8. 
8 is a default value for ``CONCURRENT_REQUESTS_PER_DOMAIN``.